app/controllers/errors_controller.rb

```diff
@@ -29,4 +30,14 @@ def login_error
       render status: 200
     end
   end
+
+  private
+
+  def message
```

Comment:
A more specific name would be helpful here, as this is specific to 404 errors.

Reply:
is `course_not_found` okay?

Reply:
It's also the message for other not found errors, so I'd go with `not_found_message`.